src/Pos/Lrc/Logic.hs

```diff
@@ -74,12 +75,12 @@ data RichmenType
     = RTUsual
     | RTDelegation (HashMap StakeholderId (HashSet StakeholderId))
 
---- | Pure version of findRichmen which uses in-memory Utxo.
+-- | Pure version of 'findRichmen' which uses in-memory Utxo.
```

Comment:
This comment is a bit outdated, because this function doesn't necessarily use Utxo.
Given that you need to force push to fix appveyor, can you update this comment?